src/test/java/net/datafaker/PassportTest.java

```diff
@@ -46,4 +46,11 @@ void testValidCanada() {
         assertThat(new Faker(new Locale("en", "CA")).passport().valid())
             .hasSize(8);
     }
+
+    @RepeatedTest(10)
+    void testValidUnitedKingdom() {
+        assertThat(new Faker(new Locale("en", "GB")).passport().valid())
+            .matches("[0-9]{9}")
+            .hasSize(9);
```

Comment:
I've done this too somewhere in the code right? It's a little bit of a duplication: the regex already is checking it's always 9 in length, so there's not really a point of checking again for the size of 9, but oh well :)